### DISCUSSION TITLE
Is Signoz really open-source or not?

### DISCUSSION THREAD

**Discussion by nitin302** [SigNoz/signoz#4231] [Q&A] [answered]
The licenses are really confusing:
https://github.com/SigNoz/signoz/blob/develop/LICENSE
https://github.com/SigNoz/signoz/blob/develop/ee/LICENSE

Is it fully open-source?

**Comment by prashant-shahi** [comment, marked as answer]
SigNoz is fully open-source. However, `ee` folder is under **SigNoz Enterprise** license, while the rest of the code is under **MIT Expat** license.

**Reply by nitin302** [reply]
https://github.com/SigNoz/signoz/tree/develop/ee - What does query-service mean? Is there a particular feature we cannot use as open source?

**Reply by nitin302** [reply]
Can we (as a company) self-host Signoz and use all of its features in production?

**Reply by nitin302** [reply]
@prashant-shahi ?

**Reply by nitin302** [reply]
https://github.com/SigNoz/signoz/discussions/2684 - I've seen this thread but it is still unclear.

**Reply by pranay01** [reply]
> Can we (as a company) self-host Signoz and use all of its features in production?

Yes, you can self host SigNoz and use it in production. Only features in community edition will be available in the open source self hosted version. Some features will be only unlocked once you take the enterprise license.

You can check the feature comparison in `Explore all features` section here - https://signoz.io/pricing/

**Reply by nitin302** [reply]
thank you @pranay01 ...

Can you point out the limits?

For example the community edition does not have **Unlimited dashboards & alerts from traces**. What is the hard limit?

**Reply by nitin302** [reply]
@pranay01 / @prashant-shahi please help answer?

**Reply by nitin302** [reply]
what is the limit for **Unlimited dashboards & alerts for logs**?

**Reply by pranay01** [reply]
We currently support 20 dashboard panels and 10 alerts in community edition

**Reply by nitin302** [reply]
@pranay01 is it the same for logs as well?

**Reply by pranay01** [reply]
The limit on number of dashboard is on the total number of panels - irrespective of the dashboard panels being based on logs or traces

**Reply by nitin302** [reply]
Just confirming:

If we self-host Signoz, then we have to follow these same steps -> https://signoz.io/docs/install/kubernetes/aws/ for setting up the community edition. We would automatically be enforced limits and limited features based on feature list here - https://signoz.io/pricing/. Is this correct?

Also, say if we were to create more than 10 alerts, we would be hard-limited and not allowed to create?

**Reply by nitin302** [reply]
@pranay01 is my above understanding correct?

**Reply by pranay01** [reply]
The restriction on alerts is only for alerts created on top of logs & traces. You can create as many alerts based on metrics as you want

**Reply by pranay01** [reply]
> We currently support 20 dashboard panels and 10 alerts in community edition

An update on this! With the demand from the community we removed any restrictions on number of dashboards and alert panels in community edition from v0.47.0 🎉

**Reply by nitin302** [reply]
thank you @pranay01. This is much appreciated. The next highest request to enable in community edition is actually SSO (at-least authentication) - https://github.com/SigNoz/signoz/issues/1188

**Reply by amolngt** [reply]
Hi all,

then what is 30 day free trial? can anyone tell which features are free?

**Reply by makeavish** [reply]
@nitin302 SSO is now enabled in community edition as well

**Reply by svyatoslavratov** [reply]
Single sign-on (SSO) is only for Google. Most of the community teams use open source solutions like Keycloak or others
